doc/OnlineDocs/explanation/solvers/pyros.rst

```diff
@@ -1061,13 +912,45 @@ Observe that the log contains the following information:
    ==============================================================================
 
 
+Observe that the log contains the following information
+(listed in order of appearance):
+
+
+* **Introductory information and disclaimer** (lines 1--19):
+  Includes the version number, author
+  information, (UTC) time at which the solver was invoked,
+  and, if available, information on the local Git branch and
+  commit hash.
+* **Summary of solver options** (lines 20--24): Enumeration of
+  specifications for optional arguments to the solver.
+* **Model component statistics** (lines 25--34):
+  Breakdown of component statistics for the user-provided model
+  and variable selection (before preprocessing).
+* **Preprocessing information** (lines 35--37):
+  Wall time required for preprocessing
+  the deterministic model and associated components,
+  i.e., standardizing model components and adding the decision rule
```

Comment:
```suggestion
  e.g., standardizing model components and adding the decision rule
```
I think you mean e.g. ("such as") here? (It's possible I am reading it incorrectly, though)